test/cmd/policy.sh

```diff
@@ -34,4 +34,39 @@ os::cmd::expect_success_and_not_text 'oc get rolebinding/cluster-admin --no-head
 
 os::cmd::expect_success 'oc policy remove-group system:unauthenticated'
 os::cmd::expect_success 'oc policy remove-user system:no-user'
+
+# adjust the cluster-admin role to check defaulting and coverage checks
+# this is done here instead of an integration test because we need to make sure the actual yaml serializations work
+workingdir=$(mktemp -d --suffix=policy)
+cp ${OS_ROOT}/test/fixtures/bootstrappolicy/old_alternate_cluster_admin.yaml ${workingdir}
+os::util::sed "s/RESOURCE_VERSION//g" ${workingdir}/old_alternate_cluster_admin.yaml
+os::cmd::expect_success "oc create -f ${workingdir}/old_alternate_cluster_admin.yaml"
+os::cmd::expect_success 'oadm policy add-cluster-role-to-user alternate-cluster-admin test-user'
```

Comment:
are we already using test-user? can we have a user just for this role to make sure no other roles are interfering with the test?